### PR TITLE
Yacreader 9

### PR DESCRIPTION
**Pull request by extrowerk**
Depends on https://github.com/haikuports/haikuports/pull/2288

### CODE REVIEW

File: media-gfx/yacreader/yacreader-9.0.0.recipe [hunk 1]
```diff
@@ -4,18 +4,21 @@ and managing your digital comic collection."
...
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
 	yacreader$secondaryArchSuffix = $portVersion
-	app:yacreader$secondaryArchSuffix = $portVersion
+	app:YACReader$secondaryArchSuffix = $portVersion
+	app:YACReaderLibrary$secondaryArchSuffix = $portVersion
+
```
Comment (alaviss) [inline media-gfx/yacreader/yacreader-9.0.0.recipe]:
```diff
-
```

File: media-gfx/yacreader/yacreader-9.0.0.recipe [hunk 2]
```diff
@@ -4,18 +4,21 @@ and managing your digital comic collection."
 HOMEPAGE="http://www.yacreader.com/"
 COPYRIGHT="Luis Ángel San Martín Rodríguez"
```
Comment (alaviss) [inline media-gfx/yacreader/yacreader-9.0.0.recipe]: The copyright year should be provided. As the author don't seem to left this information easily accessible, assuming it's the year of the latest release should be fine

```diff
-COPYRIGHT="Luis Ángel San Martín Rodríguez"
+COPYRIGHT="2018 Luis Ángel San Martín Rodríguez"

File: media-gfx/yacreader/yacreader-9.0.0.recipe [hunk 3]
```diff
@@ -29,13 +32,14 @@ REQUIRES="
 	lib:libQt5Network$secondaryArchSuffix
 	lib:libQt5OpenGL$secondaryArchSuffix
 	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libunarr$secondaryArchSuffix
 	"
 
 BUILD_PREREQUIRES="
```
Comment (alaviss) [inline media-gfx/yacreader/yacreader-9.0.0.recipe]: `BUILD_PREREQUIRES` should come after `BUILD_REQUIRES`, see the [Guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)

File: media-gfx/yacreader/yacreader-9.0.0.recipe [hunk 4]
```diff
@@ -4,18 +4,21 @@ and managing your digital comic collection."
 HOMEPAGE="http://www.yacreader.com/"
```
Comment (alaviss) [inline media-gfx/yacreader/yacreader-9.0.0.recipe]:
```diff
-HOMEPAGE="http://www.yacreader.com/"
+HOMEPAGE="https://www.yacreader.com/"

**Comment by extrowerk** [comment]
Conflict solved

**Comment by fbrosson** [comment]
Could you please remove the CRs (Carriage-Return) you have at the end of every line in your patchset?
(They prevent `haikuporter -G` from working.)

`dos2unix yacreader-9.0.0.patchset` should do the trick.

You don't want to introduce these horrible 0x0d characters in the repo, don't you? :smile:
Thanks!

**Comment by fbrosson** [comment]
Sorry, I just understood what happened: the sources **_already_** include these 0x0d characters.
Maybe we should use the exact same patch command in haikuporter, instead of one command when `-G` is used, and another if `-G` is not used.

This means you can probably ignore my previous comment.

**Pull request merged**